src/pages/play.astro

```diff
@@ -222,19 +258,28 @@ const t = useTranslations(lang)
     const footer = document.createElement('div')
     footer.classList.add('log-footer')
 
-    const disableButton = document.createElement('button')
-    disableButton.textContent = 'Disable'
-    disableButton.classList.add('btn', 'tools-secondary-btn')
-    disableButton.addEventListener('click', () => {
+    const switchButton = document.createElement('div')
+    switchButton.innerHTML = `
+      <label class="wallet-switch">
+        <span class="switch-text">Enabled</span>
+        <span class="switch-container">
+          <input type="checkbox" checked>
+          <span class="switch-slider"></span>
+        </span>
+      </label>`
+
+    switchButton.addEventListener('click', () => {
       const link = document.querySelector(
         `link[rel='monetization'][href='${href}']`
       ) as HTMLLinkElement
       link.disabled = !link.disabled
-      disableButton.textContent = link.disabled ? 'Enable' : 'Disable'
     })
 
     const removeButton = document.createElement('button')
-    removeButton.textContent = 'Remove'
+    removeButton.innerHTML = `<svg aria-hidden="true" width="20" height="20" viewBox="0 0 24 24" fill="none">
+      <path d="M7.616 20C7.168 20 6.78667 19.8426 6.472 19.528C6.15733 19.2133 6 18.8323 6 18.385V5.99998H5V4.99998H9V4.22998H15V4.99998H19V5.99998H18V18.385C18 18.845 17.846 19.2293 17.538 19.538C17.23 19.8466 16.8453 20.0006 16.384 20H7.616ZM17 5.99998H7V18.385C7 18.5643 7.05767 18.7116 7.173 18.827C7.28833 18.9423 7.436 19 7.616 19H16.385C16.5383 19 16.6793 18.936 16.808 18.808C16.9367 18.68 17.0007 18.5386 17 18.384V5.99998ZM9.808 17H10.808V7.99998H9.808V17ZM13.192 17H14.192V7.99998H13.192V17Z" fill="currentColor"/>
+      </svg>
```

Comment:
Can you extract these SVGs into a top-level constant? Or even better, add them to `src/assets` and then we can either do an `?inline` import of use `<img src>` to use them.